app/param_filters.go

```diff
@@ -74,5 +74,10 @@ func consensusParamFilter(msg sdk.Msg) error {
 		return errors.Wrapf(sdkerrors.ErrUnauthorized, "invalid validator parameters")
 	}
 
+	evidenceParams := DefaultEvidenceParams()
+	if !updateParams.Evidence.Equal(evidenceParams) {
+		return errors.Wrapf(sdkerrors.ErrUnauthorized, "invalid evidence parameters")
+	}
```

Comment:
bit of a nit

I don't like the idea of making a function called "Default..." consensus breaking. feels like a footgun. It's kinda nice as we can just change things in one spot, but idk

wdyt?

Reply:
The function is not consensus breaking. A testnode can be created with different evidence parameters. 

The param filter here just prevents the evidence params from being modified via governance. So I'm not sure what the action item is here but maybe you want this rename: `DefaultEvidenceParams()` => `EvidenceParams()`?

Reply:
>this is part of the reason why the square size integration test failed previously (yes its flakey af, but it was failing consistently with the same error indicating that evidence params were no longer equal to the defaults) afaiu

The square size integration test was failing because it attempted to modify fields in the consensus params and also set the evidence params to values that differ from the CIP-37 values.

> yeah! then what happens when we do that and try to change the maxBlockBytes? if that passes, then I'm incorrect and this isn't consensus breaking

I'll try testing it. I expect that it will be possible to change the `maxBlockBytes` as long as the evidence params remain untouched.

Reply:
https://github.com/celestiaorg/celestia-app/pull/5332/commits/0c077a88d8832efd0be4be4d3a4816791c1dae73